BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -6,28 +6,7 @@ import UIKit
 
 import BraveRewards
 
-// FIXME: Remove this struct when real data is available
-private struct UpcomingContribution {
-  let imageURL: URL?
-  let isVerified: Bool
-  let site: String
-  let attention: CGFloat
-}
-
-private let upcomingContributions = [
-  UpcomingContribution(imageURL: nil, isVerified: true, site: "myetherwallet.com", attention: 0.3),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "theverge.com", attention: 0.2),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "amazon.com", attention: 0.1),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "reddit.com", attention: 0.1),
-  UpcomingContribution(imageURL: nil, isVerified: true, site: "myetherwallet.com", attention: 0.05),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "theverge.com", attention: 0.05),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "amazon.com", attention: 0.05),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "reddit.com", attention: 0.05),
-  UpcomingContribution(imageURL: nil, isVerified: true, site: "myetherwallet.com", attention: 0.025),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "theverge.com", attention: 0.025),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "amazon.com", attention: 0.025),
-  UpcomingContribution(imageURL: nil, isVerified: false, site: "reddit.com", attention: 0.025)
-]
+typealias PublisherLoadCallback = (succes: Bool, hasMoreData: Bool)
```

Comment:
Wasn't going to bother you with this if I didn't find anything else lol

Typo: `succes` → `success`

(I'm sorry)

Reply:
Nah my bad :)

Reply:
Actually I was not using it at all. :P